.circleci/config.yml

```diff
@@ -37,6 +37,10 @@ jobs:
       - persist_to_workspace:
           root: ~/app
           paths: .
+      - run:
```

Comment:
Might be better to make this a seperate job so it can run first thing (we can choose which workflows it runs in too, we probably only need it to run for nightly workflow)